config.yaml.defaults

```diff
@@ -534,6 +534,11 @@ cross_match:
 #    transmission: [0, 1, 1, 0]
 #    description: "basic ATLAS O filter - tophat transmission"
 
+user:
+  default_role: 'Full user'
+  default_groups: ['Grandma']
```

Comment:
Let's choose something less collaboration specific.

Reply:
Bu default in skyportal there is no other group than the public group "Sitewide group" to which users are added by default thanks to this PR. So in fact i think that the best option is to leave that empty.